src/platform/user/profile/vap-svc/components/base/VAPServiceEditModal.jsx

```diff
@@ -44,7 +44,11 @@ export default class VAPServiceEditModal extends React.Component {
     // Errors returned directly from the API request (as opposed through a transaction lookup) are
     // displayed in this modal, rather than on the page. Once the modal is closed, reset the state
     // for the next time the modal is opened by removing any existing transaction request from the store.
-    if (this.props.transactionRequest && this.props.transactionRequest.error) {
+    if (
```

Comment:
since I've already made other comments on this PR, what do you think about using optional chaining for this?

```
if (this.props.transactionRequest?.error) {
  this.props.clearErrors?.()
}
```

I think that's nice, but I can definitely see an argument for including the check for `clearErrors` in the conditional itself.

Reply:
Yes, I did try this at first, but VSCode kept showing it as a "problem" - it might have just been the end of the day and my brain not at 100% 😆  - I'll try again.

Reply:
So rather than disabling an eslint message, I kept the `&&` check...

![Screen Shot 2021-07-21 at 9 29 52 AM](https://user-images.githubusercontent.com/136959/126507708-e288cbe0-bf3b-45aa-b790-6147cb6589fe.png)